### ISSUE TITLE
ServiceNow integration with Joplin

### ISSUE THREAD

**Issue by oliver2g** [cityofaustin/techstack#4376]
5/4 Meeting Notes - Integrating ServiceNow with Joplin Author Interface
- Attendees: Gerald Oliver, Tori Breitling, Jo Dwyer, Corina Preston and Melissa Regalado
- ODD is planning to track user issues for Joplin using the ServiceNow platform beginning June (end) 2020
- User issues will be related to the Joplin itself, not content
- Volume of the initial batch of users: 30-40
- Attachments (if any) will likely be image files of screenshots
- Only internal City employees will use Joplin
- info from Joplin cannot be transported to ServiceNow and pre-populated on a ticket.
- Melissa Regalado will be the ServiceNow point of contact.

Action items:
- [ ] Provide a group name and group member names to the ServiceNow team
- [x] Draft questions and logic requirements for the ServiceNow form (i.e. ticket that will be filled out by users)
- [x] Send form questions and logic to the ServiceNow team
- [ ] Test the form
- [ ] Implement the form on Joplin

**Comment by chase-vert** [comment]
Here's the design for the developers. https://share.goabstract.com/a9994d2e-6605-49b5-b97d-d27a73528c16?mode=design&sha=6733502a146b94f0c20d586d4a52a8b9a7a45512

**Comment by ablangworthy27** [comment]
I drafted questions with logic and ran them by Content Cove for feedback. I also slacked @Jo-Dwyer to find out who exactly this should be sent to. 
https://docs.google.com/document/d/1s0qG2AsUPc50dJ9yZWKeXTawXX537PAvGJm9xWtd8j8/edit#heading=h.e6j6a99iw6cr

**Comment by Jo-Dwyer** [comment]
@ablangworthy27 looks good to me and gives basic bug reporting, feedback and help request functions we're looking for. Melissa Regalado is our contact for ServiceNow. Since our last convo w/ them happened back in May, you might give some context for this form when you email her.

**Comment by ablangworthy27** [comment]
@Jo-Dwyer @toribr 
I just got done meeting with Melissa, and she is going to set the form up to only be accessible to those in CTM. As we onboard departments, we can open it up OR we can simply open it up to everyone. They seemed concerned that we might get wrongly forwarded requests that are meant for Drupal if we just open it up to everyone right now.

Next, I will reach out to Adam Parscale (sp?) in Service Now to get a knowledge article written for the form. This will be used as a reference by service desk people.

Our group in ServiceNow will be called ODD Support. We need to identify who will be the "fulfillers" and who is the support group manager. They assumed the manager is Marni, but not sure if that is true. I'm not sure if we want to identify one person from each discipline to be the fulfillers or what, but I'm open to ideas

The shared resource mailbox I gave was joplin_feedback@austintexas.gov
